### PR TITLE
Add LTK Endpoints

### PR DESCRIPTION
**Pull request by amagana3**
This PR adds the new `POST: /accessKeys` and `DEL: /IAMUser` endpoints. These endpoints will create and delete LTK users. I've made sure to _not_ include the `accessKey` & `secretKey` from the response as not to expose unwanted data. 

I've also included the appropriate tests. All is working / passing on my end. 

Good luck reviewer 👍

### CODE REVIEW

File: iam_ltk.go [hunk 1]
```diff
@@ -20,6 +21,43 @@ type GetLongTermKeysResponse struct {
...
+	AddedIAMUserToGroup bool   `json:"addedIAMUserToGroup"`
+	PartialError        bool   `json:"partialError"`
+	AccessKey           string `json:"accessKey"`
+	SecretKey           string `json:"secretKey"`
+}
+
+// LongTermKeyRequest is used to represent the request body to create or delete LTKs
+type LongTermKeyRequest struct {
+	Account     string `json:"account"`
+	Role        string `json:"role"`
```
Comment (ntangy) [inline iam_ltk.go]: I don't see role as one of the attributes in the alks api or am I looking at the wrong place

File: iam_ltk.go [hunk 2]
```diff
@@ -20,6 +21,43 @@ type GetLongTermKeysResponse struct {
...
+
+// CreateLongTermKeyResponse is the response to the CLI client
+type CreateLongTermKeyResponse struct {
+	BaseResponse
+	CreateLongTermKey
+}
+
+// DeleteLongTermKey represents the response from API
+type DeleteLongTermKey struct {
+	AddedIAMUserToGroup bool `json:"addedIAMUserToGroup"`
```
Comment (ntangy) [inline iam_ltk.go:50]: do we not need the other fields that come back from the API like "accessKey" etc

**Review by ntangy — CHANGES_REQUESTED** [review]
LGTM overall had a few questions

File: iam_ltk_test.go [hunk 3]
```diff
@@ -15,6 +15,56 @@ func (s *S) Test_GetLongTermKeys(c *C) {
...
+
+var createLongTermKeys = `
+{
+    "account": "012345678910/ALKSAdmin - AccountAlias",
+    "action": "CREATE",
+    "iamUserName": "MY_USERNAME",
+    "iamUserArn": "arn:aws:iam::012345678910:user/acct-managed/MY_USERNAME",
+    "addedIAMUserToGroup": true,
+    "partialError": false,
+	"accessKey": "thisismykey",
```
Comment (ntangy) [inline iam_ltk_test.go:63]: that indent tho

File: iam_ltk.go (hunk 2 above)
Comment (amagana3) [reply iam_ltk.go:50]: For deleting we only get back these fields, but for creating then yes `accessKey` and `secretKey` come back to reference the new LTK user

File: iam_ltk_test.go (hunk 3 above)
Comment (amagana3) [reply iam_ltk_test.go:63]: :shake-fist:

File: iam_ltk.go (hunk 1 above)
Comment (amagana3) [reply iam_ltk.go]: Wow I completely did this request body wrong, and idk how... Here's how the request body _should_ look like. Let me work on this a bit more and I'll keep you posted.

```
{
  "account":"{{accountId}}/ALKS{{role}} - {{accountAlias}}",
  "role":"{{role}}",
  "roleType": "AWS Lambda"
  "roleName": "Andrew_Test_1"
  "includeDefaultPolicy": "1"
}
```

Comment (amagana3) [reply iam_ltk.go]: WAIT! No I am looking at the wrong endpoint sorry, this is LTKs...

Create: `/rest/accessKeys`
Delete: `/rest/IAMUser`

The request bodies for both is:

```
{
  "account":"{{accountId}}/ALKS{{roleName}} - {{accountAlias}}",
  "role":"{{roleName}}",
  "iamUserName":"UNIQUE_USERNAME_HERE" 
}
```

Sorry about that @ntangy , this is correct already and shouldn't be changed. 😅

**Review by ntangy — APPROVED** [review]

**Pull request merged**